Runner support for multiple ptd files

lucylq opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #14168
* __->__ #14159

Add `std::unordered_set<std::string>` to allow for multiple data files in the runner.

Differential Revision: [D82072385](https://our.internmc.facebook.com/intern/diff/D82072385/)

lucylq committed:
Runner support for multiple ptd files

Add `std::unordered_set<std::string>` to allow for multiple data files in the runner.

Differential Revision: [D82072385](https://our.internmc.facebook.com/intern/diff/D82072385/)

[ghstack-poisoned]

kimishpatel reviewed (approved):
Review automatically exported from Phabricator review in Meta.

lucylq committed:
Update on "Runner support for multiple ptd files"


Add `std::unordered_set<std::string>` to allow for multiple data files in the runner.

Differential Revision: [D82072385](https://our.internmc.facebook.com/intern/diff/D82072385/)

[ghstack-poisoned]

lucylq committed:
Update on "Runner support for multiple ptd files"


Add `std::unordered_set<std::string>` to allow for multiple data files in the runner.

Differential Revision: [D82072385](https://our.internmc.facebook.com/intern/diff/D82072385/)

[ghstack-poisoned]

lucylq committed:
Update on "Runner support for multiple ptd files"


Add `std::unordered_set<std::string>` to allow for multiple data files in the runner.

Differential Revision: [D82072385](https://our.internmc.facebook.com/intern/diff/D82072385/)

[ghstack-poisoned]
